website/client/src/components/groups/questSidebarSection.vue

```diff
@@ -143,26 +113,43 @@
                     class="boss-health-bar"
                     :style="{width: bossHpPercent + '%'}"
                   ></div>
+                  <div
+                    class="pending-health-bar"
+                    :style="{width: pendingHpPercent + '%'}"
+                  >
+                  </div>
                 </div>
               </div>
             </div>
             <div class="row boss-details">
               <div class="col-6">
-                <span class="float-left">
-                  {{ Math.ceil(parseFloat(group.quest.progress.hp) * 100) / 100 }} / {{ parseFloat(questData.boss.hp).toFixed(2) }} <!-- eslint-disable-line max-len -->
+                <span class="float-left hp-value">
+                  <div
+                    class="svg-icon health-icon"
+                    v-html="icons.healthNoPaddingIcon"
+                  ></div>
+                  {{ Math.ceil(parseFloat(group.quest.progress.hp) * 100) / 100 }}
+                  / {{ parseFloat(questData.boss.hp) }}
+                  <strong>HP</strong>
+
                   <!-- current boss hp uses ceil so
                     you don't underestimate damage needed to end quest-->
                 </span>
               </div>
               <div
-                v-if="userIsOnQuest"
+                v-if="userIsOnQuest && user.party.quest.progress.up"
                 class="col-6"
               >
                 <!-- @TODO: Why do we not sync quest
                   progress on the group doc? Each user could have different progress.-->
-                <span
-                  class="float-right"
-                >{{ (user.party.quest.progress.up || 0) | floor(10) }} {{ $t('pendingDamageLabel') }}</span> <!-- eslint-disable-line max-len -->
+                <span class="float-right pending-value">
+                  <div
+                    class="svg-icon sword-icon"
+                    v-html="icons.swordIcon"
+                  ></div>
+                  {{ (user.party.quest.progress.up || 0) | floor(10) }}
+                  {{ $t('pendingDamageLabel') }}
+                </span> <!-- eslint-disable-line max-len -->
```

Comment:
What is this for? There's no long line...

Reply:
the previous code had it :D   I refactored it - check the previous code on this line - it needed this ignore - will be deleted now